app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -20,11 +20,11 @@
 
 .app-c-highlight-boxes__item {
   border: 1px solid $grey-2;
-  padding: $gutter-half $gutter-half $gutter;
+  padding: $gutter-half * 1.5;
   width: 100%;
   height: 100%;
-  box-sizing: border-box;
   box-shadow: 7px 7px 0 $white, 8px 8px 0 $grey-2;
+  @include box-sizing(border-box);
```

Comment:
General formatting point I picked up from Paul - he suggested to always put @includes at the start of a class. It's not hugely important though.

Reply:
Ah okay, I think I learnt to put them last - I'll bear that in mind 🙂